server/neptune/temporalworker/job/store.go

```diff
@@ -111,11 +111,6 @@ func (m *InMemoryStore) Close(ctx context.Context, jobID string, status JobStatu
 	m.lock.Lock()
 	defer m.lock.Unlock()
 
-	// Error out when job dne
-	if m.jobs[jobID] == nil {
-		return fmt.Errorf("job: %s does not exist", jobID)
-	}
-
 	// Error when job is already set to complete
 	if job := m.jobs[jobID]; job.Status == Complete {
 		return fmt.Errorf("job: %s is already complete", jobID)
```

Comment:
Won't this panic of the job id isn't in the map?

Reply:
We check if the job exists in memory before calling this method: https://github.com/lyft/atlantis/blob/b545dbcb4f258b21e133593caa76b7d24d47c50b/server/neptune/temporalworker/job/store.go#L169-L173

So, we should not get here if the job id does not exist in the map.

Reply:
Im not sure I'm following, it seems like this code will be called.

Reply:
ah i see now, either way its good to be protective I'd rather just check for it existence first and handle it in the code.